packages/runtime/src/runtime.ts

```diff
@@ -260,7 +241,7 @@ export class Runtime {
   }
 
   // creates new application transaction object and update context
```

Comment:
```suggestion
  // creates new application transaction object in the current context
  // It will overwrite the current transaction (`this.ctx.tx`)
```